pkg/webhook/cluster/mutating_webhook.go

```diff
@@ -58,18 +67,115 @@ func (a *ManagedClusterMutatingAdmissionHook) Admit(req *admissionv1beta1.Admiss
 		return status
 	}
 
+	var jsonPatches []jsonPatchOperation
+
 	// If LeaseDurationSeconds value is zero, update it to 60 by default
 	if managedCluster.Spec.LeaseDurationSeconds == 0 {
-		status.Patch = []byte(defaultLeaseDurationSecondsPatch)
-		pt := admissionv1beta1.PatchTypeJSONPatch
-		status.PatchType = &pt
+		jsonPatches = append(jsonPatches, newLeaseDurationSecondsJsonPatch(60))
```

Comment:
this can be removed this we already set default value

Reply:
Removed.